src/index.js

```diff
@@ -12,7 +16,10 @@ ReactDOM.render(
 if (module.hot) {
   module.hot.accept('./appProvider.js', () => {
     const NextAppProvider = AppProvider.default;
-    ReactDOM.render(<NextAppProvider />, document.getElementById('root'));
+    render(
+      <NextAppProvider />,
+      document.getElementById('root')
```

Comment:
this is for refreshing right?

Reply:
honestly, I don't think we actually need this. I rewrote all this in my previous company and I removed this and refreshing worked just fine.

Reply:
lets leave this to the cleanup after mainnet release. for now, if its not broke, dont fix it.